### PR TITLE
binance: update apis

### PR DESCRIPTION
**Pull request by sc0Vu**
fix: ccxt/ccxt#23253

- notice: https://developers.binance.com/docs/derivatives/change-log#2024-07-24
TODO:

- [x] update ws apis
- [x] update test

WS
```JSON
{
"fetchBalanceWs": {
                "method": "account.status, account.balance, v2/account.status, v2/account.balance"
            },
            "fetchPositionsWs": {
                "method": "account.position, v2/account.position"
            }
}
```

### CODE REVIEW

**Comment by carlosmiei** [comment]
@sc0Vu we have already this PR: https://github.com/ccxt/ccxt/pull/23225
we have to be careful to avoid a conflict

**Comment by ttodua** [comment]
@sc0Vu is this PR ready? or there are still something to be added?

**Comment by sc0Vu** [comment]
@ttodua It's ready. Lmk if you find anything.

File: ts/src/binance.ts [hunk 1]
```diff
@@ -12745,18 +12704,29 @@ export default class binance extends Exchange {
         } else {
             throw new BadRequest (this.id + ' fetchMarginModes () supports linear and inverse subTypes only');
         }
-        const assets = this.safeList (response, 'positions', []);
+        let assets = this.safeList (response, 'positions', []);
```
Comment (carlosmiei) [inline ts/src/binance.ts:12707]: hum `crossed` is not unified 
image

**Pull request merged**